### PR TITLE
Add Quality Metrics to viz notebook

### PR DESCRIPTION
**Pull request by CBroz1**
### Submission checklist

- [X] Reviewed all `Files changed` prior to selecting `Create Pull Request`.
- [ ] Ran and passed all pytests, and ran the pipeline for features not covered by tests.: *Not applicable*
- [ ] Ran pre-commit checks, including black, flake8, etc: *Not yet implemented*
- [X] Updated the changelog and if making a new release, updated the version.py.
- [ ] Updated documentation, including concepts.md, tutorials.md, and relevant docstrings: *Not applicable*
- [X] After merge, I will push a new tag to upstream (test* prefix if no new release).

### Submission details

In this pull request, I make the following changes:
- Notebook/py_script: Add QC plots. Jupytext sync
- Version bump in changelog
- Version bump version.py to reflect previous changes with listed release date
- Analysis.py - fix typing issue

### CODE REVIEW

File: workflow_array_ephys/version.py [hunk 1]
```diff
@@ -2,4 +2,4 @@
 Package metadata
 Update the Docker image tag in `docker-compose.yaml` to match
 """
-__version__ = "0.2.4"
+__version__ = "0.2.5"
```
Comment (kushalbakshi) [inline workflow_array_ephys/version.py:5]:
```suggestion
__version__ = "0.2.6"
```

Comment (kushalbakshi) [reply workflow_array_ephys/version.py:5]: I believe this matches the CHANGELOG version

**Review by kushalbakshi — COMMENTED** [review]
Just need to get the versions aligned and this will be good

**Comment by CBroz1** [comment]
Thanks @kushalbakshi - Good catch. Yes, the repo should reflect the unreleased number

**Review by kushalbakshi — APPROVED** [review]

**Pull request merged**